conductr_cli/conduct_load.py

```diff
@@ -223,3 +232,22 @@ def continue_logging(monitor):
 
 def string_io(input_text):
     return io.StringIO(input_text)
+
+
+def cleanup_old_bundles(cache_dir, bundle_file_name, excluded):
+    bundle_name_parts = bundle_file_name.split('-')
+    # Remove digest, keeping only bundle name and compatible version
+    bundle_name = '-'.join(bundle_name_parts[:-1])
+
+    # List of bundle files having the same name and compatibility version, sorted from oldest to latest.
+    # This list excludes the file specified as `excluded`, and normally the `excluded` file is the recently loaded
+    # bundle.
+    older_bundle_files = sorted([
+        file
+        for file in glob.glob('{}/*.zip'.format(cache_dir))
+        if os.path.isfile(file) and os.path.basename(file).startswith(bundle_name) and not file == excluded
+    ], key=lambda f: os.path.getmtime(f))
+
+    bundle_files_to_delete = older_bundle_files[:(-1 * KEEP_BUNDLE_VERSIONS)]
+    for file in bundle_files_to_delete:
+        os.remove(file)
```

Comment:
I suspect that `os.remove` does not work on Windows. We have a pending issue when deleting bundle configuration files: https://github.com/typesafehub/conductr-cli/issues/175

So before merging the PR we need to verify if the solution works on Windows as well.